plutus-playground-server/test/Playground/UsecasesSpec.hs

```diff
@@ -174,12 +175,12 @@ gameTest =
     testGroup
         "game"
         [ compilationChecks game
-        , testCase "should keep the funds" $
-          evaluate (mkEvaluation [lock w2 "abcde" twoAda, AddBlocks 1, guess w1 "ade", AddBlocks 1]) >>=
-          hasFundsDistribution
-              [ mkSimulatorWallet w1 tenAda
-              , mkSimulatorWallet w2 (adaValueOf 8)
-              ]
+        -- , testCase "should keep the funds" $
```

Comment:
Are these the tests that will be enabled again later on?

Reply:
Yes